src/WebJobs.Script/Workers/Http/Configuration/HttpWorkerOptions.cs

```diff
@@ -15,8 +15,19 @@ public class HttpWorkerOptions
 
         public int Port { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the host will forward the request to the worker process.
+        /// </summary>
+        /// <remarks>
+        /// The host will rebuild the initial invocation HTTP Request and send the copy to the worker process.
+        /// </remarks>
         public bool EnableForwardingHttpRequest { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the host will proxy the invocation HTTP request to the worker process.
+        /// </summary>
+        public bool EnableProxyingHttpRequest { get; set; }
```

Comment:
This came up in R2D.

Doesn't look like `HttpWorkerOptions` implements `IOptionsFormatter`. Can we please follow up with a change to make sure we have telemetry indicading when this feature is used?